src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculator, FigureDrawing {
+    private String color;
+
+    public Figure() {
+
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public void draw() {
+        System.out.print("Figure: ");
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }

```

Comment:
delete this method

Reply:
why? I asked this question before
![image](https://github.com/mate-academy/jv-oop-advanced/assets/71705897/6691ab38-a0a7-42a1-a420-cee8531adb7a)

Reply:
Figure has to be abstract. Such classes could have an abstract methods